### PR TITLE
FAI-11476: Add Xray converter

### PR DESCRIPTION
**Pull request by chalenge**
## Description
Converters for 
- Tests
- Plans
- Executions
- TestPlanTests
- TestRuns

## Type of change
- [ ] Bug fix
- [X] New feature
- [ ] Breaking change

### CODE REVIEW

File: destinations/airbyte-faros-destination/src/converters/xray/common.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import {Converter} from '../converter';
+
+export interface ModelEnumType {
+  category: string;
+  detail: string;
+}
+
+export abstract class XrayConverter extends Converter {
+  source = 'Xray';
+  taskSource = 'Jira';
```
Comment (matiaslcoulougian) [inline destinations/airbyte-faros-destination/src/converters/xray/common.ts:12]: Whats the use of this property?

**Review by matiaslcoulougian — APPROVED** [review]
✅

**Comment by matiaslcoulougian** [comment]
Also, remove unused imports and update index spec test snapshot

**Comment by chalenge** [comment]
> Also, remove unused imports and update index spec test snapshot

Noted, we should fix the linter to catch these and fail the test

File: destinations/airbyte-faros-destination/src/converters/xray/common.ts (hunk 1 above)
Comment (chalenge) [reply destinations/airbyte-faros-destination/src/converters/xray/common.ts:12]: Each model is actually a real task in Jira hence we associate it [e.g.](https://github.com/faros-ai/airbyte-connectors/pull/1512/files#diff-cecb6e9215b1d22296d783a4e5e0292f4878724f9134e51a8a6589a138575d68R26)

**Pull request merged**